plugins/bigquery/dbt/include/bigquery/macros/adapters.sql

```diff
@@ -27,41 +27,50 @@
 {%- endmacro -%}
 
 {% macro bigquery_table_options(persist_docs, temporary, kms_key_name) %}
-  {% set opts = {} %}
+  {% set opts = {} -%}
 
-  {% set description = get_relation_comment(persist_docs, model) %}
+  {%- set description = get_relation_comment(persist_docs, model) -%}
   {%- if description is not none -%}
-    {% do opts.update({'description': "'" ~ description ~ "'"}) %}
-  {% endif %}
-  {% if temporary %}
+    {%- do opts.update({'description': "'" ~ description ~ "'"}) -%}
+  {%- endif -%}
+  {%- if temporary -%}
     {% do opts.update({'expiration_timestamp': 'TIMESTAMP_ADD(CURRENT_TIMESTAMP(), INTERVAL 12 hour)'}) %}
-  {% endif %}
-  {% if kms_key_name %}
-    {% do opts.update({'kms_key_name': "'" ~ kms_key_name ~ "'"}) %}
-  {% endif %}
+  {%- endif -%}
+  {%- if kms_key_name -%}
+    {%- do opts.update({'kms_key_name': "'" ~ kms_key_name ~ "'"}) -%}
+  {%- endif -%}
+  {%- if labels -%}
+    {%- set label_list = [] -%}
+    {%- for key in labels -%}
```

Comment:
how about something like?
```suggestion
    {%- for label, value in labels.items() -%}
```

Reply:
This is less ugly :)